rust/experimental/server/src/app.rs

```diff
@@ -308,6 +308,7 @@ pub struct WritingViewContext {
 pub struct ReadingViewContext {
     pub uid: PartitionedUId,
     pub reading_options: ReadingOptions,
+    pub serialized_expected_task_ids_bitmap: Bytes,
```

Comment:
I will prefer marking like this.


`pub serialized_expected_task_ids_bitmap: Option<Treemap>,`

Reply:
sure, let me update it